src/macro.jl

```diff
@@ -34,6 +34,7 @@ macro resumable(expr::Expr)
 
   # The function that executes a step of the finite state machine
   func_def = splitdef(expr)
+  @debug func_def[:body]
```

Comment:
I like these `@debug` statements, please do not remove them after you are done. The overhead of the logger is fine in these "compilation" steps.